<launch>



    <node pkg="kros_utils" type="image_saver_stereo" name="image_saver_stereo" output="screen">
        <param name="BAG_FULL_FILENAME" value="/home/manohar/workspace/outdoor_nav_Danjiao/sensor_calibration/fisheye_calibration_0421.bag" />

        <param name="LEFT_IMAGE_TOPIC" value="/camera/fisheye1/image_raw/compressed" />
        <param name="RIGHT_IMAGE_TOPIC" value="/camera/fisheye2/image_raw/compressed" />

        <param name="SAVE_PATH" value="/home/manohar/try/sync_stereo_pairs/" />
        <param name="LEFT_PREFIX" value="leftimage"/>
        <param name="RIGHT_PREFIX" value="rightimage"/>

        <param name="POSTFIX" value=".png" />

        <param name="is_compressed_stream" value="true" />
        <param name="save_every_in_sec" value="0.5" />

        <!-- not in use -->
        <!-- <param name="INDEX" value="number" /> -->
        <param name="INDEX" value="timestamp" /> 

    </node>

</launch>
